tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/profile/descartes_default_plan_profile.h

```diff
@@ -53,13 +59,19 @@ class DescartesDefaultPlanProfile : public DescartesPlanProfile<FloatType>
   DescartesDefaultPlanProfile& operator=(DescartesDefaultPlanProfile&&) noexcept = default;
   DescartesDefaultPlanProfile(const tinyxml2::XMLElement& xml_element);
 
+#ifndef SWIG
   PoseSamplerFn target_pose_sampler = [](const Eigen::Isometry3d& tool_pose) {
```

Comment:
Would it be better to move this to the cpp constructor?

Reply:
Same parser failure as above